src/modules/dashboard/components/ActionsPage/ActionsPage.css

```diff
@@ -5,6 +5,23 @@
   background-color: white;
 }
 
+.recoveryTag {
+  padding: 4.5px 8px;
```

Comment:
The text would look better "centered" vertically if the top and bottom padding would just be `4px`

```suggestion
  padding: 4px 8px;
```

![Screenshot from 2020-12-18 16-03-36](https://user-images.githubusercontent.com/1193222/102623158-bf6e6300-414a-11eb-8e74-ade0d2289f2b.png)